mlflow/tracking/__init__.py

```diff
@@ -411,3 +411,39 @@ def _get_git_commit(path):
         return commit
     except (InvalidGitRepositoryError, GitCommandNotFound, ValueError, NoSuchPathError):
         return None
+
+
+def _get_git_url(uri):
```

Comment:
I'd rename this to `_get_git_uri_if_present` to make it clearer, and also update the doc to say that this returns the path `git_uri#sub_directory` if the URI passed is a local path that's part of a Git repo, or returns the original URI otherwise.

Reply:
Done